### PR TITLE
Kubectl expose command accepts a filename param

### PR DESCRIPTION
**Pull request by feihujiang**
address part of issue #10628

### CODE REVIEW

**Comment by feihujiang** [comment]
@JanetKuo @caesarxuchao Please review this PR.

**Comment by brendandburns** [comment]
LGTM.

File: pkg/kubectl/cmd/expose.go [hunk 1]
```diff
@@ -37,6 +37,9 @@ re-use the labels from the resource it exposes.`
 	expose_example = `// Creates a service for a replicated nginx, which serves on port 80 and connects to the containers on port 8000.
 $ kubectl expose rc nginx --port=80 --target-port=8000
 
+// Creates a service for a replicated nginx from nginx-controller.yaml, which serves on port 80 and connects to the containers on port 8000.
```
Comment (JanetKuo) [inline pkg/kubectl/cmd/expose.go]: Like other kubectl reviews, change `//` to `#` and claify that the file here is used only to get the identity of the resource.

File: pkg/kubectl/cmd/expose.go [hunk 2]
```diff
@@ -71,11 +74,14 @@ func NewCmdExposeService(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().String("overrides", "", "An inline JSON override for the generated object. If this is non-empty, it is used to override the generated object. Requires that the object supply a valid apiVersion field.")
 	cmd.Flags().String("name", "", "The name for the newly created object.")
 	cmd.Flags().String("session-affinity", "", "If non-empty, set the session affinity for the service to this; legal values: 'None', 'ClientIP'")
+
+	usage := "Filename, directory, or URL to a file containing the resource to expose"
```
Comment (JanetKuo) [inline pkg/kubectl/cmd/expose.go]: ditto

**Comment by piosz** [comment]
Shippable is failing while trying to verify gendocs. Rebase should fix the problem. Could you do it?

**Comment by feihujiang** [comment]
@piosz OK

**Comment by piosz** [comment]
@k8s-bot test this please

**Comment by piosz** [comment]
Thanks for the change!

**Pull request merged**